samples/samples-aspnet/embedded-sign-in-widget/README.md

```diff
@@ -0,0 +1,93 @@
+# ASP.NET Core & Embedded-Widget Login Page Example
+
+## Introduction
+> :grey_exclamation: The use of this Sample uses an SDK that requires usage of the Okta Identity Engine. 
+This functionality is in general availability but is being gradually rolled out to customers. If you want
+to request to gain access to the Okta Identity Engine, please reach out to your account manager. If you 
+do not have an account manager, please reach out to oie@okta.com for more information.
+
+This Sample Application will show you the best practices for integrating Authentication by embedding the 
+Sign In Widget into your application. The Sign In Widget is powered by [Okta's Identity Engine](https://
+developer.okta.com/docs/concepts/ie-intro/) and will adjust your user experience based on policies. 
+Once integrated, you will be able to utilize all the features of Okta's Sign In Widget in your application.
+
+For information and guides on how to build your app with this sample, please take a look at the [{FRAMEWORK} 
+guides for Embedded Sign In Widget Authentication](link to DevDoc SBS guide)
+
+## Installation & Running The App
+
+### Prerequisites
+
+Before running this sample, you will need the following:
+
+* An Okta Developer Account, you can sign up for one at https://developer.okta.com/signup/.
+* An Okta Application, configured for Web mode. This is done from the Okta Developer Console; find instructions [here][OIDC Web Application Setup Instructions].  When following the wizard, use the default properties.  They are designed to work with our sample applications.
+
+### Clone the repository
+
+```git clone https://github.com/okta/samples-aspnetcore.git```
+
+### Run the web application
+
+Run the example with your preferred tool, see [Run the web application from Visual Studio](#run-the-web-application-from-visual-studio) or [Run the web application from dotnet CLI](#run-the-web-application-from-dotnet-cli).
+
+> **NOTE:** This sample is using ASP.NET Core 3.1 which enforces HTTPS. This is a recommended practice for web applications. See [Enforce HTTPS in ASP.NET Core] for more details.
```

Comment:
I thought we were using .NET 4.8?

Reply:
Good point I copied this from my core sample.  I'll fix it.